main.c

```diff
@@ -64,6 +64,9 @@ int main(void)
 	MSS_TIM1_enable_irq();
 	NVIC_SetPriority(Timer1_IRQn, 1);
 
+	/* For trial */
+	delay_test();
```

Comment:
I'm guessing this is test code and will be removed - right?

Reply:
yup